mapadroid/worker/WorkerMITM.py

```diff
@@ -191,7 +191,16 @@ def _check_for_data_content(self, latest_data, proto_to_wait_for: ProtoIdentifie
         if latest_proto_data is None:
             return LatestReceivedType.UNDEFINED, data_found
         latest_proto = latest_proto_data.get("payload")
-        key_to_check: str = "wild_pokemon" if mode in ["mon_mitm", "iv_mitm"] else "forts"
+
+        if mode in ["mon_mitm", "iv_mitm"]:
+            settings = self._mapping_manager.routemanager_get_settings(self._routemanager_name)
+            if settings.get("nearby_cell_mode", False):
+                key_to_check = "nearby_pokemon"
```

Comment:
Introduce enum / constants for proto keys

Reply:
Unsure what you mean here. `key_to_check` is set to str since it's later being passed to https://github.com/Map-A-Droid/MAD/blob/master/mapadroid/worker/MITMBase.py#L443 which only accepts str.

Reply:
both "nearby_cell_mode" and "nearby_pokemon" could be extracted to a constant in e.g. madGlobals.py. Just so that the keys could easily be changed in a single place rather than a potential high number.
I am aware that it is not the case way too often in current master's code... Hoping to reduce all the string/dict accesses on my orm_asyncio branch